driver-core/src/main/com/mongodb/internal/operation/FindOperation.java

```diff
@@ -470,7 +471,7 @@ private boolean isAwaitData() {
         return cursorType == CursorType.TailableAwait;
     }
 
-    private CommandReadTransformer<BsonDocument, AggregateResponseBatchCursor<T>> transformer() {
+    private CommandReadTransformer<BsonDocument, QueryBatchCursor<T>> transformer() {
```

Comment:
Using the implementation name rather than the interface - bringing it inline with the rest of the code base. Same as done in the AggregateOperationImpl